pkg/completion/completion.go

```diff
@@ -91,3 +91,15 @@ func AutocompleteSubgidName(cmd *cobra.Command, args []string, toComplete string
 func AutocompleteSubuidName(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
 	return autocompleteSubIDName("/etc/subuid")
 }
+
+// AutocompleteArch - Autocomplete architectures supported by container engines
+func AutocompleteArch(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
+	completions := []string{"aarch64", "i686", "ppc64le", "s390x", "x86_64"}
+	return completions, cobra.ShellCompDirectiveNoFileComp
+}
+
+// AutocompleteOS - Autocomplete OS supported by container engines
+func AutocompleteOS(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
+	completions := []string{"linux", "windows"}
```

Comment:
Might want to add more os.

Reply:
Sure, but what ones actually support containers?

Reply:
Only if those OS support OCI Containers, which I don't believe they do.